packages/core/scripts/local/getMedianHistoricalPrice.js

```diff
@@ -43,6 +43,7 @@ async function getMedianHistoricalPrice(callback) {
     let priceFeedConfig = {
       // Empirically, Cryptowatch API only returns data up to ~4 days back.
       lookback: 345600,
+      priceFeedDecimals: 18, // Ensure all prices come out as 18-decimal denominated so the fromWei conversion works at the end.
```

Comment:
This is neccessary because of the `fromWei` below right?

Reply:
Yep.